### PR TITLE
91 move makefile

### LINKED ISSUE #91: Do you really need a Makefile with a python package?
There are not a lot of python packages in Debian that use a Makefile to run tests. Though I've only looked at three or four, I can't seem to find one. Is this a "pythonic" approach to use a Makefile for tests? I see a different approach in Debian that has the power to declare separate dependencies for tests as well.

### PR DESCRIPTION
**Pull request by hesa**
Closes #91 
Removed makefile

### CODE REVIEW

**Comment by priv-kweihmann** [comment]
Just my 2 cents - the changes of this PR are okay to me, **but** I can't agree with the commit messages.

I think we should enforce the following template

```
component: summary

why was it changed

Closes #reference
```

which makes this here for example

```
Makefile: remove file

as it's confusing and not maintained. 
We are doing everything with setuptools, so there is not need for
an extra Makefile

Closes #91
```

File: devel/release.sh [hunk 1]
```diff
@@ -116,14 +120,12 @@ inform Checkout $VERSION
 git checkout $VERSION
 if [ $? -ne 0 ]; then echo "Failed checking our $VERSION"; exit 2; fi
 
-inform Build
-make
```
Comment (jens-erdmann) [inline devel/release.sh:120]: This seems to be a mix up with a previous commit. Shouldn't this be part of "remove confusing makefile"?

Comment (hesa) [reply devel/release.sh:120]: Hmm, I guess so.

Comment (hesa) [reply devel/release.sh:120]: Yeah...it's a sign I did not run test tests properly

Comment (hesa) [reply devel/release.sh:120]: Perhaps a discussion on the relevance of this file (`devel/release.sh`) is needed?

**Comment by jens-erdmann** [comment]
@hesa is this still relevant?

**Comment by hesa** [comment]
my beers on closing this down, and hide it so people can't see my mess ;)

**Comment by priv-kweihmann** [comment]
> my beers on closing this down, and hide it so people can't see my mess ;)

apparently only class 1 beer ;-) - okay so then let's close it

**Pull request closed without merging**